ecs.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use PhpCsFixer\Fixer\Import\NoUnusedImportsFixer;
+use Symplify\EasyCodingStandard\Config\ECSConfig;
+use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
+
+return function (ECSConfig $ecsConfig): void {
+    $ecsConfig->paths([
+        __DIR__ . '/app',
+        __DIR__ . '/plugins',
+        __DIR__ . '/tests',
+    ]);
+
+    // this way you add a single rule
+    $ecsConfig->rules([
+        NoUnusedImportsFixer::class,
+    ]);
+
+    // this way you can add sets - group of rules
+    $ecsConfig->sets([
+        // run and fix, one by one
+        // SetList::SPACES,
+        // SetList::ARRAY,
+        SetList::DOCBLOCK,
+        // SetList::NAMESPACES,
+        // SetList::COMMENTS,
+        // SetList::PSR_12,
+    ]);
+};
```

Comment:
I've added this config temporarily, so it's easier to clean the coding standard issues.